src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -34,7 +34,7 @@
     public static final String ncrcatPath = "/usr/bin/ncrcat";
     public static final String ncdpqPath = "/usr/bin/ncpdq";
 
-    private final String geoserverUrl;
+    private UrlIndexInterface urlIndexInterface = null;
```

Comment:
It'll probably be `FeatureSourceIndexReader` - just so we are consistent. What do you think?